networkx/algorithms/flow/tests/test_maxflow.py

```diff
@@ -72,7 +72,7 @@ def validate_cuts(G, s, t, solnValue, partition, capacity, flow_func):
         assert not nx.is_strongly_connected(H), errmsg
 
 
-def compare_flows_and_cuts(G, s, t, solnFlows, solnValue, capacity="capacity"):
+def compare_flows_and_cuts(G, s, t, solnValue, capacity="capacity"):
```

Comment:
I don't particularly like these "helper" functions for the tests as IMO the indirection makes the testing logic harder to follow (hence missing this completely unused parameter).

Instead of removing it however, is there away that validation can be added for the `solnFlows` inputs? It seems like this was something that was intended to be checked but was just missed. Is it straightforward to add the check?

Reply:
Makes sense to try to incorporate the `solnFlows` parameter. Based on my initial look, some of the in-place solutions don't pass as expected. Will dig in more to see if there's a good way to add the flows back in.

Reply:
I got around to taking another look at this and I suspect (without having proven) that the issue is that the "solutions" represent only a single particular solution out of a set of possible solutions.

In this case I'm in favor of removing these cases as proposed!